docs/user/user/server-knobs/metrics/product-integration.md

```diff
@@ -55,13 +55,13 @@ lazy.NimbusFeatures.yourNimbusFeatureName.onUpdate(() => {
 
 ### Integration Option 1:
 
-Glean provides a general Nimbus feature that can be used for configuration of metrics named `glean`. Simply select the `glean` feature along with your Nimbus feature from the Experimenter UI when configuring the experiment or rollout (see https://experimenter.info for more information on multi-feature experiments).
+Glean provides a general Nimbus feature that can be used for configuration of metrics named `glean`. Simply select the `glean` feature along with your Nimbus feature from the Experimenter UI when configuring the experiment or rollout (see [Nimbus] documentation for more information on multi-feature experiments).
 
-The `glean` Nimbus feature requires the `gleanMetricConfiguration` variable to be used to provide the require metric configuration. The format of the configuration is a map of the fully qualified metric name (category.name) of the metric to a boolean value representing whether the metric is enabled. If a metric is omitted from this map, it will default to the value found in the metrics.yaml. An example configuration for the `glean` feature can be found on the [Experimenter Configuration](./experimenter-configuration.md) page.
+The `glean` Nimbus feature requires the `gleanMetricConfiguration` variable to be used to provide the require metric configuration. The format of the configuration is a map of the fully qualified metric name (category.name) of the metric to a boolean value representing whether the metric is enabled. If a metric is omitted from this map, it will default to the value found in the metrics.yaml. An example configuration for the `glean` feature can be found on the [Experimenter Configuration] page.
```

Comment:
```suggestion
The `glean` Nimbus feature requires the `gleanMetricConfiguration` variable to be used to provide the required metric configuration. The format of the configuration is a map of the fully qualified metric name (category.name) of the metric to a boolean value representing whether the metric is enabled. If a metric is omitted from this map, it will default to the value found in the metrics.yaml. An example configuration for the `glean` feature can be found on the [Experimenter Configuration] page.
```